PostHog/Surveys/PostHogSurveyIntegration.swift

```diff
@@ -499,7 +523,6 @@
         /// Sends a `survey dismissed` event to PostHog instance
         private func sendSurveyDismissedEvent(survey: PostHogSurvey) {
             let additionalProperties: [String: Any] = [
-                "$survey_questions": survey.questions.map(\.question),
```

Comment:
survey dismissed should not have `$survey_questions`

Reply:
mmm i dont follow because our APIs only query for survey sent events when getting responses, how is that useful?

Reply:
I think makes sense, we can drop it. When we implement partial responses for mobile SDKs (sometime in the future) we'll have access to all responses regardless. Will remove from the JS sdk too